django_mongodb/operations.py

```diff
@@ -3,13 +3,21 @@
 
 from bson.decimal128 import Decimal128
 from django.conf import settings
+from django.db import NotSupportedError
 from django.db.backends.base.operations import BaseDatabaseOperations
+from django.db.models.expressions import Combinable
 from django.utils import timezone
 from django.utils.regex_helper import _lazy_re_compile
 
 
 class DatabaseOperations(BaseDatabaseOperations):
     compiler_module = "django_mongodb.compiler"
+    combine_operators = {
+        Combinable.ADD: "add",
```

Comment:
given that this is mongo's operations, shouldn't we put the actually mongo's operators (like $add, $multiply.. and so on)?

Reply:
I followed the pattern elsewhere, e.g. https://github.com/mongodb-labs/django-mongodb/blob/6e5995e04bdd4d81fee8af5d1f68301e07701126/django_mongodb/functions.py#L36-L57

Reply:
Well, It is just a matter of personal preference to keep the $ here or in the functions.